README.md

```diff
@@ -2,42 +2,1104 @@
+| `contentEncoding`      | No       | The content encoding of the field.                                                                                                       | ✔️                   | -              |
+| `contentMediaType`     | No       | The content media type of the field.                                                                                                     | ✔️                   | -              |
+| `precision`            | No       | The precision of a numeric field.                                                                                                        | ✔️                   | -              |
+| `scale`                | No       | The scale of a numeric field.                                                                                                            | ✔️                   | -              |
+| `encoding`             | No       | The encoding type of the field.                                                                                                          | ✔️                   | -              |
+| `minimum`              | No       | The minimum value of a numeric field.                                                                                                    | ✔️                   | -              |
+| `exclusiveMinimum`     | No       | Specifies if the minimum value is exclusive.                                                                                             | ✔️                   | -              |
+| `maximum`              | No       | The maximum value of a numeric field.                                                                                                    | ✔️                   | -              |
+| `exclusiveMaximum`     | No       | Specifies if the maximum value is exclusive.                                                                                             | ✔️                   | -              |
+
+## Mapping in Blindata
+
+The observer supports the use of two specifications: Datastore API and Async API. For Datastore API the schema format
+supported is JSON, while for Async API is AVRO.
+
+### Systems
+
+The 'platform' field within the 'promises' field of the port in the descriptor is used to extract the name and system
+technology of the system to be created in Blindata.
+
+| Data product descriptor field | Blindata Field | Mandatory |
+|-------------------------------|----------------|-----------|
+| `promises.platform`           | `system.name`  | -         |
+
+To extract the name and technology for association, two regular expressions must be defined within the Blindata
+configurations, as shown below.
+
+```yaml
+blindata:
+  systemNameRegex: optional regex to extract system name from schema (value optional)
+  systemTechnologyRegex: optional regex to extract system technology from schema (value optional)
+```
+
+### DatastoreApi
+
+##### Physical Entity
```

Comment:
Modificare in "From JSONSchema to Physical Entities".
Aggiornare l'indice.
Verificare il livello del titolo (4 o 5 #? bisogna rimanere coerenti con i successivi)